server/src/main/java/org/schoellerfamily/pathfinderhelper/datamodel/Character.java

```diff
@@ -0,0 +1,337 @@
+package org.schoellerfamily.pathfinderhelper.datamodel;
+
+import java.util.ArrayList;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class Character {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String name;
+    private Long playerid;
+    private Integer strength;
+    private Integer dexterity;
+    private Integer constitution;
+    private Integer intelligence;
+    private Integer wisdom;
+    private Integer charisma;
+    //private PathfinderClassLevel levels[];
+    //private Race race;
+    private String gender;
+    private Integer age;
+    private Integer height;//inches
+    private Integer weight;
+    private String description;
+    private Integer baseattackbonus;
+    private Integer size;
+    private Integer maximumhealth;
+    private Integer initiative;
+    private Integer fortitude;
+    private Integer reflex;
+    private Integer will;
+    private Integer combatmaneuverbonus;
+    private Integer combatmaneuverdefense;
+    private Integer landspeed;
+    private Integer swimspeed;
+    private Integer flightspeed;
+    private String flightmaneuverability;
+    private Integer burrowspeed;
+    //private Item inventory[];
+    //private SkillRank skillranks[];
+    //private HashMap<String, HashMap<Integer, ArrayList<Spell>>> spellsknown;
+    //private HashMap<String, HashMap<Integer, ArrayList<Spell>>> spellsprepared;
+    //private HashMap<String, Integer[10]> spellslots;
+    //private ArrayList<Ability> abilities;
+    private ArrayList<String> passives;
+    
+    public Character() {
+    		id = null;
+    		name = null;
+    		setPlayerid(null);
+    		strength = new Integer(10);
```

Comment:
Should these attributes have some validation of range? If so, do you throw exceptions or simply coerce into range?